views/tasks/view.php

```diff
@@ -9,70 +9,72 @@
 
 $this->title = "Задание: $task->title";
 ?>
-<div class="left-column">
-    <div class="head-wrapper">
-        <h3 class="head-main"><?= $task->title; ?></h3>
-        <p class="price price--big"><?= $task->budget; ?> ₽</p>
-    </div>
-    <p class="task-description"><?= $task->description; ?></p>
-    <a href="#" class="button button--blue">Откликнуться на задание</a>
-    <div class="task-map">
-        <img class="map" src="<?= Url::to(['@web/img/map.png']); ?>"  width="725" height="346" alt="Новый арбат, 23, к. 1">
-        <p class="map-address town">Москва</p>
-        <p class="map-address">Новый арбат, 23, к. 1</p>
-    </div>
-    <h4 class="head-regular">Отклики на задание</h4>
-<?php if ($responses): ?>
-    <?php foreach ($responses as $response): ?>
-        <div class="response-card">
-            <img class="customer-photo" src="<?= $response->user->avatar; ?>" width="146" height="156" alt="Фото заказчиков">
-            <div class="feedback-wrapper">
-                <a href="<?= Url::to(['/user/view/', 'id' => $response->user->id]); ?>" class="link link--block link--big"><?= $response->user->name; ?></a>
-                <div class="response-wrapper">
-                    <div class="stars-rating small"><?= StarWidget::getStars(count($response->user->reviews)); ?></div><br>
-                    <p class="reviews"><?= count($response->user->reviews); ?> отзыва</p>
+<main class="main-content container">
+    <div class="left-column">
+        <div class="head-wrapper">
+            <h3 class="head-main"><?= $task->title; ?></h3>
+            <p class="price price--big"><?= $task->budget; ?> ₽</p>
+        </div>
+        <p class="task-description"><?= $task->description; ?></p>
+        <a href="#" class="button button--blue">Откликнуться на задание</a>
+        <div class="task-map">
+            <img class="map" src="<?= Url::to(['@web/img/map.png']); ?>"  width="725" height="346" alt="Новый арбат, 23, к. 1">
```

Comment:
можно без алиаса `@web`:

```suggestion
            <img class="map" src="<?= Url::to(['/img/map.png']); ?>"  width="725" height="346" alt="Новый арбат, 23, к. 1">
```